src/main/resources/db/migration/common/V1_217__drop_courthouse_region_cra_id.sql

```diff
@@ -0,0 +1,5 @@
+-- V47
+ALTER TABLE courthouse_region_ae DROP COLUMN cra_id;
+CREATE UNIQUE INDEX courthouse_region_ae_pk ON courthouse_region_ae(cth_id,reg_id);
+
+
```

Comment:
Do we also need to drop the sequence?

```
DROP SEQUENCE IF EXISTS cra_seq;
```

Reply:
yes its being done in V1_219__v48_without_transcription.sql

Reply:
I've just seen this down below, maybe move it into here?

Reply:
I have kept the changes in chronological order that's why they are split. this change was missed in v47 and the dropping of the sequence was in v48.